src/cmd/ksh93/bltins/math.c

```diff
@@ -113,7 +113,7 @@ static Sfdouble_t local_nexttoward(int type_1, Sfdouble_t arg_1, int type_2, Sfd
 }
 
 static int local_signbit(Sfdouble_t a1) {
-    return signbit(a1);  //!OCLINT(constant conditional operator)
+    return !!signbit(a1);  //!OCLINT(constant conditional operator)
```

Comment:
Up to you whether you leave it this way but I'm not a fan of this trick. It confuses people and causes lint tools to complain about the double negation. I prefer the less confusing `signbit(a1) != 0`.